projects/rocprofiler-sdk/source/lib/python/rocpd/libpyrocpd.cpp

```diff
@@ -419,8 +418,112 @@ PYBIND11_MODULE(libpyrocpd, pyrocpd)
+                                    WHEN 'User Time' THEN 'CPU User Time'
+                                    WHEN 'Virtual Memory Usage' THEN 'CPU Virtual Memory Usage'
+                                    WHEN 'thread_context_switch' THEN 'Thread Context Switches'
+                                    WHEN 'thread_cpu_time' THEN 'Thread CPU Time'
+                                    WHEN 'thread_page_fault' THEN 'Thread Page Faults'
+                                    WHEN 'thread_peak_memory' THEN 'Thread Peak Memory Usage'
+                                    ELSE PMC_I.description
+                                END AS short_description,
+                                A.absolute_index AS agent_abs_index,
+                                R.event_id,
+                                E.stack_id AS stack_id,
+                                E.parent_stack_id AS parent_stack_id,
+                                E.correlation_id AS corr_id,
+                                E.extdata AS extdata,
+                                E.call_stack AS call_stack,
+                                E.line_info AS line_info
+                            FROM
+                                `rocpd_sample` R
+                                INNER JOIN `rocpd_track` T ON T.id = R.track_id
+                                AND T.guid = R.guid
+                                INNER JOIN `rocpd_event` E ON E.id = R.event_id
+                                AND E.guid = R.guid
+                                INNER JOIN `rocpd_info_process` P ON P.id = T.pid
+                                AND P.guid = T.guid
+                                INNER JOIN `rocpd_string` SN ON SN.id = T.name_id
+                                AND SN.guid = T.guid
+                                INNER JOIN `rocpd_string` CN ON CN.id = E.category_id
+                                AND CN.guid = T.guid
+                                INNER JOIN `rocpd_info_pmc` PMC_I ON SN.string =
+                                CASE CN.string
+                                    WHEN 'amd_smi' THEN PMC_I.name || ' [' ||
+                                       (SELECT type_index
+                                        FROM `rocpd_info_agent` A
+                                        WHERE A.id = PMC_I.agent_id) || ']'
+                                    ELSE  PMC_I.name
+                                END
+                                AND PMC_I.guid = T.guid
+                                INNER JOIN `rocpd_info_agent` A ON A.id = PMC_I.agent_id
+                                AND A.guid = T.guid
+                                LEFT OUTER JOIN `rocpd_info_thread` TH ON TH.id = T.tid
+                                AND TH.guid = T.guid;
+                        )";
+
+                execute_raw_sql_statements(conn, create_samples_view);
+
+                const std::string create_pmc_event_view = R"(
+                            CREATE TEMP VIEW
```

Comment:
```suggestion
                            CREATE TEMP VIEW IF NOT EXISTS
```
Same fix for multiple DBs required.

Reply:
Done